src/main/java/randoop/generation/GrtFuzzing.java

```diff
@@ -0,0 +1,480 @@
+   *     (incremented).
+   * @return a sequence with the fuzzing statement appended at the end
+   */
+  private static Sequence appendFuzzingOperation(
+      Sequence sequence,
+      Executable fuzzingOperation,
+      Type outputType,
+      AtomicInteger fuzzStatementCount) {
+    CallableOperation callableOperation = createCallableOperation(fuzzingOperation);
+    NonParameterizedType declaringType =
+        new NonParameterizedType(fuzzingOperation.getDeclaringClass());
+    List<Type> inputTypeList = getInputTypeList(fuzzingOperation, declaringType);
+    TypeTuple inputType = new TypeTuple(inputTypeList);
+    TypedOperation typedOperation =
+        new TypedClassOperation(callableOperation, declaringType, inputType, outputType);
+    List<Integer> inputIndex = calculateInputIndices(sequence.size(), inputTypeList.size());
+    fuzzStatementCount.addAndGet(inputIndex.size());
+    List<Sequence> sequenceList = Collections.singletonList(sequence);
+    return Sequence.createSequence(typedOperation, sequenceList, inputIndex);
+  }
+
+  /**
+   * Create a callable operation given the executable.
+   *
+   * @param executable the executable to create the callable operation for
+   * @return a callable operation for the given executable
+   */
+  private static CallableOperation createCallableOperation(Executable executable) {
+    if (executable instanceof Method) {
+      return new MethodCall((Method) executable);
+    } else {
+      return new ConstructorCall((Constructor<?>) executable);
+    }
+  }
+
+  /**
+   * Returns the output type of the given executable.
+   *
+   * @param executable the executable to get the output type of
+   * @return the output type of the given executable
+   */
+  private static Type getOutputType(Executable executable) {
+    return Type.forType(executable.getAnnotatedReturnType().getType());
+  }
+
+  /**
+   * Get the list of input types for the given executable. Note: This method doesn't handle cases
+   * where the executable is a constructor of a non-static inner class. This is because all
+   * executable objects passed to this method are known fuzzing operations, which none are part of
+   * an inner class.
+   *
+   * @param executable the executable to get the input types of
+   * @param declaringType the type that declares the given executable
+   * @return the list of input types for the given executable
+   */
+  private static List<Type> getInputTypeList(
+      Executable executable, NonParameterizedType declaringType) {
+    List<Type> inputTypeList = new ArrayList<>();
+    if (!Modifier.isStatic(executable.getModifiers()) && executable instanceof Method) {
+      inputTypeList.add(declaringType);
+    }
+    for (Class<?> cls : executable.getParameterTypes()) {
+      inputTypeList.add(Type.forClass(cls));
+    }
+    return inputTypeList;
+  }
+
+  /**
+   * Calculate the indices of the input statements for the fuzzing operation. This method is called
```

Comment:
The documentation speaks of "the fuzzing operation" and "the sequence", but it's not clear what these refer to, from reading the Javadoc in isolation.